torch/distributed/_tensor/examples/comm_mode_features_example.py

```diff
@@ -130,17 +130,17 @@ def test_MLPStacked_distributed_sharding_display(self) -> None:
     def test_MLP_module_tracing(self) -> None:
         """
         Example code to demonstrate CommModeDebug's module level tracing using a MLP model.
-        Prints a table of module level collective tracing information
+        Prints a table of module level collective tracing information and logs table to output.txt
 
         Expected Output
         Global
-        c10d_functional.all_reduce: 1
+        *c10d_functional.all_reduce: 1
         MLPModule
-            c10d_functional.all_reduce: 1
+            *c10d_functional.all_reduce: 1
             MLPModule.net1
             MLPModule.relu
             MLPModule.net2
-            c10d_functional.all_reduce: 1
+            *c10d_functional.all_reduce: 1
```

Comment:
this indentation seems different from the one in the PR summary?

Reply:
Yeah I think the one in summary is the newer behavior. Need update in-code comment.